.gitlab-ci.hpc-spack.yml

```diff
@@ -0,0 +1,28 @@
+include:
+  - project: hpc/gitlab-pipelines
+    file: spack-build-components.gitlab-ci.yml
+
+default:
+  tags: []
+
+spack-setup:
+  tags: [bb5_map]
+spack-build:
+  tags: [bb5_map]
+  extends: .spack_setup_ccache
+  stage: test
+  variables:
+    SALLOC_ACCOUNT: proj3
+    SBATCH_ACCOUNT: proj3
+    SLURM_ACCOUNT: proj3
+    bb5_account: proj3
+    SPACK_PACKAGE_NAME: brayns
+    SPACK_PACKAGE_COMPILER: gcc
+    SPACK_PACKAGE_SPEC: ^ospray@1.8.5%intel +brion
+    DATADIR: /tmp 
+  
+  # TEST: TO BE REMOVED BEFORE MERGE!!!
+  rules:
```

Comment:
FWIW, these rules should then be located on the `worklfow` level or even in the parent pipeline, conditionally creating the subpipeline. Preferably the latter, as subpipelines will fail if there's no job created for them.

Reply:
Thanks, changed!